README.md

```diff
@@ -104,6 +104,43 @@ the vast majority of work involved in implementing `inert` is a necessary pre-cu
 so by implementing `inert` first,
 implementers may get useful functionality into the hands of developers sooner while still laying the groundwork for one or both of these more complex APIs.
 
+### Use cases
+
+- **Temporarily offscreen/hidden content**
+
+  As discussed in the [article](https://robdodson.me/building-better-accessibility-primitives/#problem2disablingtabindex),
+  there are a range of circumstances in which case it's desirable to add content to the DOM to be rendered but remain offscreen.
+
+  In these cases, without `inert`, authors are forced to choose between
+  an accessible experience for keyboard and assistive technology users,
+  or the factors (such as performance) which make offscreen rendering desirable -
+  or, performing all the contortions necessary to keep the offscreen content functionally "inert".
+
+  These cases include:
+
+  + rendering content, such as a menu, offscreen, before having it animate on-screen;
+  + similarly, for content like a menu which may be repeatedly shown to the user,
+  avoiding re-rendering this content each time;
+  + a carousel or other type of content cycler (such as a "tweet cycler")
+  which hides non-current items by placing them at a lower z-index than the active item;
```

Comment:
we didn't do z-index but we did opacity cross-fades and that created a problem because you can't just set something to 'visibility: hidden' right away, you have to fade and listen for transitionend but there's no transitionstart event so you end up with code in two places to inert/un-inert stuff